tokio/CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# 1.30.0 (August 6, 2023)
+
+tokio: bump MSRV to 1.63 (#5887)
+
+### Fixed
+- runtime: fix flaky test `wake_while_rt_is_dropping` (#5905)
+
+### Changed
+
+- tokio: reduce LLVM code generation (#5859)
+- poll: Do not clear readiness on short read/writes. (#5881)
+- sync: make `const_new` methods always available (#5885)
+- sync: avoid false sharing in mpsc channel (#5829)  
+### Added
+
+- sync: add `broadcast::Sender::new` (#5824)
+- net: implement `UCred` for espidf (#5868)
+- fs: add `File::options()` (#5869)
+- time: implement extra reset variants for `Interval` (#5878)
+- rt: pop at least one task from inject queue (#5908)
+
+### Removed
+
+- tokio: removed unused `tokio_*` cfgs (#5890)
+- the entire build.rs is removed so compiling Tokio should be faster
+
+### Documented
+
+- sync: mention lagging in docs for `broadcast::send` (#5820)
+- runtime: expand on sharing runtime docs (#5858)
+- io: use vec in example for `AsyncReadExt::read_exact` (#5863)
+- time: mark `Sleep` as `!Unpin` in docs (#5916)
+- process: fix `raw_arg` not showing up in docs (#5865)
+
+### Unstable
+- rt(alt): fix a number of concurrency bugs (#5907)
+- rt(alt): track which workers are idle. (#5886)
+- rt: add runtime ID (#5864)
+- rt: initial implementation of new threaded runtime (#5823)
```

Comment:
IMO users only need to know a new alternative runtime is merged, they can check each PR if they are curious on what work has been done.

```suggestion
- rt: add runtime ID (#5864)
- rt(alt): initial implementation of new threaded runtime (#5823 #5886 #5907)
```

Reply:
updated!